src/reputation.h

```diff
@@ -28,7 +28,8 @@
 
 #include "host.h"
 
-#define SREP_MAX_CATS 60
+#define SREP_MAX_CATS 120
```

Comment:
the idea of the 60 number was that the 'SReputation' struct became 64 bytes this way. With 120 it would be 124, a weird number.

But I will reject this change for other reasons. The expansion of the available space should be more significant, I'm thinking more about allowing thousands and using something like int16_t for the values. Or maybe event 32bits. Until we figure out a design for that I don't want to change the format.